test/Utils.Tests.ps1

```diff
@@ -0,0 +1,122 @@
+. $PSScriptRoot\..\Utils.ps1
+
+Describe 'Utils Function Tests' {
+    Context 'Add-ImportModuleToProfile Tests' {
+        BeforeAll {
+           $newLine = [System.Environment]::NewLine
+        }
+        It 'Creates profile file if it does not exist' {
+            $profilePath = [System.IO.Path]::GetTempFileName()
+            Remove-Item -LiteralPath $profilePath
+            Test-Path -LiteralPath $profilePath | Should Be $false
+            try {
+                $scriptRoot = Split-Path $profilePath -Parent
+                Add-ImportModuleToProfile $profilePath $scriptRoot
+                Test-Path -LiteralPath $profilePath | Should Be $true
+                Get-FileEncoding $profilePath | Should Be 'ascii'
+                $content = Get-Content $profilePath
+                $content.Count | Should Be 1
+                @($content)[0] | Should BeExactly "Import-Module '$scriptRoot\posh-git.psd1'"
+            }
+            finally {
+                Remove-Item $profilePath -ErrorAction SilentlyContinue

```

Comment:
Heh, I started with that.  Then wrote a few tests that didn't need $profilePath and removed Before/AfterEach.  Then I moved those tests to a different Context.  Fixed.